### DISCUSSION TITLE
TBL setup

### DISCUSSION THREAD

**Discussion by yumilili** [xcompact3d/Incompact3d#262] [Q&A] [answered]
- For boundary layer DNS setup, is the Reynolds number (Re) in the input file based on Re=U_infty * delta_in / nu (U_infty = 1, delta_in =1, delta_in is the inlet Blasius delta_99 thickness)?

- How to choose the boundary layer tripping parameters based on my mesh since it is mesh-dependent.
A_tr = 0.1783125 
xs_tr_tbl=1.402033
ys_tr_tbl=0.350508
ts_tr_tbl=1.402033
x0_tr_tbl=3.505082

Thanks!

**Comment by slaizet** [comment, marked as answer]
Hi,
Yes for the definition of the Reynolds number.
Regarding the tripping, I suggest you have a look at the paper from where the tripping subroutine was designed: Schlatter, P., & Örlü, R. (2012). Turbulent boundary layers at moderate Reynolds numbers: inflow length and tripping effects. Journal of Fluid Mechanics, 710, 5-34.
I think the only option is test and trial errors!
Best,
Sylvain

**Reply by yumilili** [reply]
Many thanks!
